src/window/hotkey_duplicate.c

```diff
@@ -0,0 +1,90 @@
+#include "hotkey_duplicate.h"
+
+#include "core/image_group.h"
+#include "core/string.h"
+#include "graphics/graphics.h"
+#include "graphics/image_button.h"
+#include "graphics/panel.h"
+#include "graphics/text.h"
+#include "graphics/window.h"
+#include "input/input.h"
+
+static void button_ok(int param1, int param2);
+
+static image_button buttons[] = {
+    {223, 140, 39, 26, IB_NORMAL, GROUP_OK_CANCEL_SCROLL_BUTTONS, 0, button_ok, button_none, 1, 0, 1},
+};
+
+static struct {
+    const uint8_t *title;
+    const uint8_t *message;
+    const uint8_t *action_name;
+} data;
+
+static int init(translation_key title, translation_key message, const uint8_t * action_name)
+{
+    // QUESTION
+    // do we need this here? copy paste from WINDOW_PLAIN_MESSAGE_DIALOG
```

Comment:
Answer: if we expand `plain_message_dialog` instead, we don't need it here because this file won't be necessary.
If we keep this dialog, I think it's also not needed, since it's a specific dialog, not a generic one like the `plain_message_dialog`.